common/culture/cultures/wc_grummle.txt

```diff
@@ -6,6 +6,11 @@
 	language = language_mogu
 	martial_custom = martial_custom_male_only
 	traditions = {
+		tradition_esteemed_hospitality
+		tradition_pacifism
+		tradition_zealous_people
+		tradition_himalayan_settlers
```

Comment:
You need to uncomment it then and replace because bonuses are the same.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/174496651-57271c03-d836-457f-bb09-93c3150fd475.png)

</details>

Reply:
The bonuses are different. 
For `Sacred Mountains`:
```
	province_modifier = {
		desert_mountains_tax_mult = 0.05
		mountains_tax_mult = 0.05
	}
```
For `Himalayan Settlers`:
```
	character_modifier = {
		mountains_development_growth_factor = 0.2
	}
	province_modifier = {
		mountains_construction_gold_cost = -0.15
		mountains_holding_construction_gold_cost = -0.15
	}
```

Reply:
> The bonuses are different. For `Sacred Mountains`:

True. I guess Paradox locked it because otherwise it would allow to stack too many mountain bonuses thus I would do the same for balance reason.